modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -19,16 +19,26 @@ If you miss this step, the second boot will fail.
 For example, to configure DHCP networking, identify `ip=dhcp`
 or set static networking when you add parameters to the kernel command line.
 
-. Generate the thumbprint. Install the clevis package, it is not already
-installed, and generate a thumbprint
-from the Tang server. Replace the value of `url` with the Tang server URL:
+. Generate the thumbprint.
```

Comment:
It is correct to break this into two steps. However, you should cut "Generate the thumbprint." and start the step with: "Install the clevis package, if it is not already installed:" Notice that I added the "if" which was missing.